docker-compose.yml

```diff
@@ -18,3 +18,9 @@ services:
     ports:
```

Comment:
1. instead of wait hosts might be better do to a health check to ensure that it's able to accept the connectio n and not just booted up.
2. Might as well update the version whule we're here
3. why not use the aws infromation 
4. wonder if it's worth making the volumes read only and also adding an additional one a named volume for persistence
5. no network? might be worth adding one for insolation purposes
6. coould add the image service running type ie node:20 etc. 
7. I also presume that this is going to be dependant on the s3 and sqs now starting
8.

Reply:
Though I agree with what you've said, I'm also keen to make these changes minimal and related to the particular ticket.

The local development needs a bit of an overhaul, but for the purposes of getting serverless running and enabling debug feature in code we're really just missing the sqs container.

Happy if we add more tickets after all the cert gen changes are merged in? (otherwise it would likely mean changing things here, then again later anyway).